res/documentation/creating new release.md

```diff
@@ -7,7 +7,7 @@ A new release is a good time to check if all dependencies are still up to date.
 - **Update outdated dependencies** as indicated by Android Studio in the `build.gradle`. For each to-be-updated dependency, review the changelog (see [release notes for androidx libraries](https://developer.android.com/jetpack/androidx/releases/appcompat)) - are there any (breaking) changes that may affect us? Test it.
 - **Update available languages**. Check [on POEditor](https://poeditor.com/projects/view?id=97843) if any translation not included yet reached a certain threshold of completion (~80%) to be included or if any already included translation has been abandoned (only ~50% translated). If necessary, adapt `bcp47ExportLanguages` in `app/build.gradle.kts` and mention the change in the changelog.
 - **Update presets version**. Check if `presetsVersion` and `nsiVersion` in the `build.gradle` still point to the latest version of the [iD presets](https://github.com/openstreetmap/id-tagging-schema/releases) and [name suggestion index](https://github.com/osmlab/name-suggestion-index/tags) respectively, check if the schema has any incompatible changes and update. 
-- **Update main contributor listing and other manual credit handling**. In general credits are updated automatically (see below), but [main contributors](app/src/main/res/raw/credits_main.yml) and list of ignored commits ([`skipCommit`](build.gradle.kts)) are handled manually.
+- **Update main contributor listing and other manual credit handling**. In general credits are updated automatically (see below), but [main contributors](../../app/src/main/res/raw/credits_main.yml) and list of ignored commits ([`skipCommit`](../../build.gradle.kts)) are handled manually.
```

Comment:
y?

Reply:
links were 404 otherwise

Reply:
is no absolute link possible?

Reply:
Sure, but that would also break when the repo is moved/renamed, whatever, so it always has disadvantages.

hmm https://docs.github.com/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files … let me try out sth.

Reply:
Though they recommend relative links there, I've changed it now. It works on GitHub.com fine at least.